parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetOutputFormat.java

```diff
@@ -210,6 +214,29 @@ public static boolean getEnableDictionary(JobContext jobContext) {
     return getEnableDictionary(getConfiguration(jobContext));
   }
 
+  public static Map<String, Long> getBloomFilterColumnExpectedNDVs(Configuration conf) {
+    Map<String, Long> kv = new HashMap<>();
+    String columnNamesConf = conf.get(BLOOM_FILTER_COLUMN_NAMES);
+    String expectedNDVsConf = conf.get(BLOOM_FILTER_EXPECTED_NDV);
+
+    if (columnNamesConf == null || expectedNDVsConf == null) {
+      return kv;
+    }
+
+    String[] columnNames = columnNamesConf.split(",");
+    String[] expectedNDVs = expectedNDVsConf.split(",");
+
+    if (columnNames.length == expectedNDVs.length) {
+      for (int i = 0; i < columnNames.length; i++) {
+        kv.put(columnNames[i], Long.getLong(expectedNDVs[i]));
```

Comment:
@chenjunjiedada 
I have a small question about `BLOOM_FILTER_EXPECTED_NDV`.  
Why do we get the **system property** after setting `parquet.bloom.filter.expected.ndv`? Wouldn't it be better if we just parse the string with `Long.parseLong()`?

Reply:
@garawalid,  Usually the compute engine, such as hive, spark, etc., set the properties for parquet so we may not able to get the string directly other than from configuration.

Reply:
@chenjunjiedada Thanks for the clarification.
In fact, when I pass `parquet.bloom.filter.column.names` and  `parquet.bloom.filter.expected.ndv` this way :  
```java
    // Configuration conf = new Configuration();
    conf.set("parquet.bloom.filter.column.names", "content,line");
    conf.set("parquet.bloom.filter.expected.ndv","1000,200");
```

I got `kv` as follows `"line" -> null` and `"content" -> null`. 

You can reproduce this behavior by adding that configuration to `testReadWrite()` in `TestInputOutputFormat.java`. 

Do you think this is normal? In my case, I changed `Long.getLong(expectedNDVs[i])` by `Long.parseLong(expectedNDVs[i])` to build the `kv` HashMap.